cmd/mirror/event_store.go

```diff
@@ -95,10 +96,10 @@ func copyEvents(ctx context.Context, source, dest *db.DB, bulkSize uint32) {
 	maxPosition, err := writeMigrationStart(ctx, sourceES, migrationID, dest.DatabaseName())
 	logging.OnError(err).Fatal("unable to write migration started event")
 
-	logging.WithFields("from", previousMigration.Position, "to", maxPosition).Info("start event migration")
+	logging.WithFields("from", previousMigration.Position.String(), "to", maxPosition.String()).Info("start event migration")
```

Comment:
We don't actually need to call the `String()` method here. If a `fmt.Stringer` is passed to the logger, the method should be called when the logline is printed, when the required log-level is passed for example.

https://go.dev/play/p/tQ1Z906t8EG